test/test_distributions.py

```diff
@@ -116,6 +124,9 @@ def _mvn_to_scipy(loc, cov, prec, tril):
     T(dist.MultivariateNormal, np.arange(6, dtype=np.float32).reshape((3, 2)), None, None,
       np.array([[1., 0.], [0., 1.]])),
     T(dist.MultivariateNormal, 0., None, np.broadcast_to(np.identity(3), (2, 3, 3)), None),
+    T(dist.LowRankMultivariateNormal, np.zeros(2), np.array([1, 0]), np.diag(np.array([1, 1]))),
```

Comment:
`cov_diag` is a vector, so I think `np.array([1, 1])` is enough.

`cov_factor` is a matrix, so I guess you will want to define it as: `np.array([[1], [0]])`